readme.md

```diff
@@ -125,17 +88,51 @@ A nice example of this is [mdx-deck][], a great way to create slides with MDX.
 *   [Next.js plugin](https://mdxjs.com/getting-started/next)
 *   [Gatsby plugin](https://mdxjs.com/getting-started/gatsby)
 
-## Related
-
-See related projects in the [MDX specification][spec].
-
 ## Authors
 
 *   [John Otander][john] ([@4lpine][4lpine]) – [Compositor][] + [Clearbit][]
 *   [Tim Neutkens][tim] ([@timneutkens][timneutkens]) – [ZEIT][]
 *   [Guillermo Rauch][guillermo] ([@rauchg][rauchg]) – [ZEIT][]
 *   [Brent Jackson][brent] ([@jxnblk][jxnblk]) – [Compositor][]
 
+## Sponsors
```

Comment:
I wonder if it should be above Authors 🤔

Reply:
Yeah, I think you're right!